diff --git a/dist-cjs/is-crt-available.js b/dist-cjs/is-crt-available.js
index 4d794148f7cf0eb8f359b1a9199967b74aad8168..37bd07699265f31394a55a7812bd17cff517f679 100644
--- a/dist-cjs/is-crt-available.js
+++ b/dist-cjs/is-crt-available.js
@@ -3,9 +3,6 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.isCrtAvailable = void 0;
 const isCrtAvailable = () => {
     try {
-        if (typeof require === "function" && typeof module !== "undefined" && module.require && require("aws-crt")) {
-            return ["md/crt-avail"];
-        }
         return null;
     }
     catch (e) {