diff --git a/node_modules/@aws-sdk/util-user-agent-node/dist-cjs/is-crt-available.js b/node_modules/@aws-sdk/util-user-agent-node/dist-cjs/is-crt-available.js
index 4d79414..37bd076 100644
--- a/node_modules/@aws-sdk/util-user-agent-node/dist-cjs/is-crt-available.js
+++ b/node_modules/@aws-sdk/util-user-agent-node/dist-cjs/is-crt-available.js
@@ -3,9 +3,6 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.isCrtAvailable = void 0;
 const isCrtAvailable = () => {
     try {
-        if (typeof require === "function" && typeof module !== "undefined" && module.require && require("aws-crt")) {
-            return ["md/crt-avail"];
-        }
         return null;
     }
     catch (e) {
