diff --git a/dist-cjs/SignatureV4MultiRegion.js b/dist-cjs/SignatureV4MultiRegion.js
index 1bad845e8f3f539963dfc4e367698c86b8e928ff..1a88afa01670b50c2b9dfc9673febec695b5b7a7 100644
--- a/dist-cjs/SignatureV4MultiRegion.js
+++ b/dist-cjs/SignatureV4MultiRegion.js
@@ -27,7 +27,7 @@ class SignatureV4MultiRegion {
         if (!this.sigv4aSigner) {
             let CrtSignerV4;
             try {
-                CrtSignerV4 = typeof require === "function" && require("@aws-sdk/signature-v4-crt").CrtSignerV4;
+                CrtSignerV4 = undefined
                 if (typeof CrtSignerV4 !== "function")
                     throw new Error();
             }