diff --git a/node_modules/@walletconnect/keyvaluestorage/dist/cjs/node-js/db.js b/node_modules/@walletconnect/keyvaluestorage/dist/cjs/node-js/db.js
index 3401caf..7559257 100644
--- a/node_modules/@walletconnect/keyvaluestorage/dist/cjs/node-js/db.js
+++ b/node_modules/@walletconnect/keyvaluestorage/dist/cjs/node-js/db.js
@@ -1,12 +1,7 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 function importLokijs() {
-    try {
-        return require("lokijs");
-    }
-    catch (e) {
-        throw new Error(`To use WalletConnect server side, you'll need to install the "lokijs" dependency. If you are seeing this error during a build / in an SSR environment, you can add "lokijs" as a devDependency to make this error go away.`);
-    }
+    throw new Error(`To use WalletConnect server side, you'll need to install the "lokijs" dependency. If you are seeing this error during a build / in an SSR environment, you can add "lokijs" as a devDependency to make this error go away.`);
 }
 let Lokijs;
 class Db {
