diff --git a/lib/index.es.js b/lib/index.es.js
index 4852f7cbac354d002a601212332187487deb3afa..a813786e3c9bfb36b64d0f75fbd064f3882bffce 100644
--- a/lib/index.es.js
+++ b/lib/index.es.js
@@ -155,9 +155,7 @@ FetchError.prototype.constructor = FetchError;
 FetchError.prototype.name = 'FetchError';
 
 let convert;
-try {
-	convert = require('encoding').convert;
-} catch (e) {}
+
 
 const INTERNALS = Symbol('Body internals');
 
diff --git a/lib/index.js b/lib/index.js
index e5b04f107f765db04265c7635d3ca1d86b146fb8..934eb00f3f186d38b95076893883a20258d88c1c 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -159,9 +159,7 @@ FetchError.prototype.constructor = FetchError;
 FetchError.prototype.name = 'FetchError';
 
 let convert;
-try {
-	convert = require('encoding').convert;
-} catch (e) {}
+
 
 const INTERNALS = Symbol('Body internals');
 
diff --git a/lib/index.mjs b/lib/index.mjs
index 49ee05ecf06d31a5168795ced5d795c93a0226e3..5cb6549e516c098260406cae78ddb6828be97164 100644
--- a/lib/index.mjs
+++ b/lib/index.mjs
@@ -153,9 +153,7 @@ FetchError.prototype.constructor = FetchError;
 FetchError.prototype.name = 'FetchError';
 
 let convert;
-try {
-	convert = require('encoding').convert;
-} catch (e) {}
+
 
 const INTERNALS = Symbol('Body internals');
 