diff --git a/lib/index.es.js b/lib/index.es.js
index 79d717be9ae56d455f7f5c7890178bc4bb65da9c..6acc11733872938c4410faaf98f98afde6975fe1 100644
--- a/lib/index.es.js
+++ b/lib/index.es.js
@@ -155,9 +155,7 @@ FetchError.prototype.constructor = FetchError;
 FetchError.prototype.name = 'FetchError';
 
 let convert;
-try {
-	convert = require('encoding').convert;
-} catch (e) {}
+
 
 const INTERNALS = Symbol('Body internals');
 
diff --git a/lib/index.js b/lib/index.js
index 337d6e52e5fc62c26f1f46ac115505fa8e327b5c..53184f792c50c07de66f4516ae9ace72dbabbb7b 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -159,9 +159,7 @@ FetchError.prototype.constructor = FetchError;
 FetchError.prototype.name = 'FetchError';
 
 let convert;
-try {
-	convert = require('encoding').convert;
-} catch (e) {}
+
 
 const INTERNALS = Symbol('Body internals');
 
diff --git a/lib/index.mjs b/lib/index.mjs
index ace669fd0fee2f3a0e6ca0d34e46c60377da7c78..a3288be005f0c7b242c6986299995a0f34c66765 100644
--- a/lib/index.mjs
+++ b/lib/index.mjs
@@ -153,9 +153,7 @@ FetchError.prototype.constructor = FetchError;
 FetchError.prototype.name = 'FetchError';
 
 let convert;
-try {
-	convert = require('encoding').convert;
-} catch (e) {}
+
 
 const INTERNALS = Symbol('Body internals');
 