diff --git a/node_modules/three/build/three.js b/node_modules/three/build/three.js
index d305cc3..350008e 100644
--- a/node_modules/three/build/three.js
+++ b/node_modules/three/build/three.js
@@ -32295,13 +32295,6 @@
 			}
 		}));
 	}
-	if (typeof window !== 'undefined') {
-		if (window.__THREE__) {
-			console.warn('WARNING: Multiple instances of Three.js being imported.');
-		} else {
-			window.__THREE__ = REVISION;
-		}
-	}

 	exports.ACESFilmicToneMapping = ACESFilmicToneMapping;
 	exports.AddEquation = AddEquation;
